packages/blade/src/components/Checkbox/_decisions/decisions.md

```diff
@@ -0,0 +1,263 @@
+| hasError           | `boolean`  | No       | `false`   | Control whether the checkbox is invalid or not.                                                                                                          |
+| isRequired          | `boolean`  | No       | `false`   | Control whether the checkbox is required or not.                                                                                                         |
+
+### Examples:
+
+#### Basic
+
+```tsx
+// Label text
+<Checkbox>is blade awesome?</Checkbox>
+// Helper text
+<Checkbox helpText="Hint: yes">is blade awesome?</Checkbox>
+// uncontrolled
+<Checkbox defaultChecked={true}>is blade awesome?</Checkbox>
+// controlled
+<Checkbox isChecked={true | false} onChange={isChecked => {}}>is blade awesome?</Checkbox>
+```
+
+#### Validations
+
+And `hasError` will pass `aria-invalid` attributes to indicate invalid state.
+
+```tsx
+<Checkbox hasError={true | false}>is blade awesome?</Checkbox>
+```
+
+## `CheckboxGroup` API
+
+| Prop          | Type                        | Required | Default     | Description                                                                                                                        |
+| ------------- | --------------------------- | -------- | ----------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| children      | `React.ReactNode`             | Yes       | `undefined` | Accepts multiple Checkboxes                                                                                                        |
+| label         | `string`                    | Yes       | `undefined` | The label of the group                                                                                                             |
+| labelPosition | `top, left`                 | No       | `top`       | The position of the rendered label                                                                                                 |
+| helpText    | `string`                    | No       | `undefined` | The helper text to be rendered                                                                                                     |
+| errorText    | `string`                    | No       | `undefined` | The error text to be rendered                                                                                                     |
+| defaultValue  | `string[]`                  | No       | `[]`        | The initial value of the checkbox group                                                                                            |
+| value         | `string[]`                  | No       | `[]`        | The value of the checkbox, if present will act as controlled component                                                             |
```

Comment:
Should these always accept an array or can it take a string when only 1 value needs to be passed? 🤔

Reply:
Always array. `string | string[]` will just make things complicated.

Reply:
We can do only string[] array now since for single checkboxes we can now use individual checkbox.